src/openapi/paths.yaml

```
@@ -0,0 +1,69 @@
+openapi: 3.0.4
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)